hyperspy/drawing/utils.py

```diff
@@ -458,137 +470,141 @@ def plot_images(images,
-            Values should be supplied as used in pyplot.subplots_adjust(),
-            and can be:
-                'left', 'bottom', 'right', 'top', 'wspace' (width),
-                and 'hspace' (height)
-        tight_layout : bool, optional
-            If true, hyperspy will attempt to improve image placement in
-            figure using matplotlib's tight_layout
-            If false, repositioning images inside the figure will be left as
-            an exercise for the user.
-        aspect : str or numeric, optional
-            If 'auto', aspect ratio is auto determined, subject to min_asp.
-            If 'square', image will be forced onto square display.
-            If 'equal', aspect ratio of 1 will be enforced.
-            If float (or int/long), given value will be used.
-        min_asp : float, optional
-            Minimum aspect ratio to be used when plotting images
-        namefrac_thresh : float, optional
-            Threshold to use for auto-labeling. This parameter controls how
-            much of the titles must be the same for the auto-shortening of
-            labels to activate. Can vary from 0 to 1. Smaller values
-            encourage shortening of titles by auto-labeling, while larger
-            values will require more overlap in titles before activing the
-            auto-label code.
-        fig : mpl figure, optional
-            If set, the images will be plotted to an existing MPL figure
-        vmin, vmax : scalar or list of scalar, optional, default: None
-            If list of scalar, the length should match the number of images to
-            show.
-            A list of scalar is not compatible with a single colorbar.
-            See vmin, vmax of matplotlib.imshow() for more details.
-        *args, **kwargs, optional
-            Additional arguments passed to matplotlib.imshow()
-
-        Returns
-        -------
-        axes_list : list
-            a list of subplot axes that hold the images
-
-        See Also
-        --------
-        plot_spectra : Plotting of multiple spectra
-        plot_signals : Plotting of multiple signals
-        plot_histograms : Compare signal histograms
-
-        Notes
-        -----
-        `interpolation` is a useful parameter to provide as a keyword
-        argument to control how the space between pixels is interpolated. A
-        value of ``'nearest'`` will cause no interpolation between pixels.
-
-        `tight_layout` is known to be quite brittle, so an option is provided
-        to disable it. Turn this option off if output is not as expected,
-        or try adjusting `label`, `labelwrap`, or `per_row`
+    Extra keyword arguments are passed to `matplotlib.figure`.
+
+    Parameters
+    ----------
+    images : list of Signal2D or BaseSignal
+        `images` should be a list of Signals to plot. For `BaseSignal` with
+        navigation dimensions of 2, the signal will be tranposed, which is
```

Comment:
What about: "for signals with navigation dimension 2 and signal dimension 0...".

I would remove the remark regarding EDS maps since this feature is useful in many other cases.